InteractionBase/src/InteractionBasePlugin.cpp

```diff
@@ -39,6 +39,8 @@
 
 #include "vis/CommandPrompt.h"
 #include "vis/TextAndDescription.h"
+#include "vis/VViewSwitcherEntry.h"
```

Comment:
We add this and the next include, but don't use them. Are you missing something, or should we remove these two includes?